composer/tools/composer_dags.md

```diff
@@ -0,0 +1,83 @@
+# Composer DAGs Pausing/Unpausing script
+
+This document describes usage of composer_dags.py script.
+
+Purpose of the script it to provide a tool to pause or unpause all the DAGs 
+(except airflow_monitoring) of a Composer environment.
+
+## Prerequisites
+
+1.  [Make sure you are authorized](https://cloud.google.com/sdk/gcloud/reference/auth/login)
+    through `gcloud auth login` before invoking the script . The script requires
+    [permissions to access the Composer environment](https://cloud.google.com/composer/docs/how-to/access-control)
+    (roles/composer.environmentAndStorageObjectAdmin role is sufficient) and its
+    GKE cluster (container.*).
+
+1.  Make sure that you can access GKE cluster of Composer environment through
+    kubectl (e.g. if you are not able to successfully execute
+    `kubectl get pods`, the script will fail). "Unable to connect to the server:
+    dial tcp [IP ADDRESS]:443: connect: connection timed out" reported by
+    kubectl indicates that kubectl can not connect to GKE cluster.
+    [Refer to the documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#cloud_shell)
+    about setting up access to private GKE cluster if you have private IP environment.
+
+1.  Supported environment versions: all Composer versions 
+
+1.  The script depends on Python 3.6 (or newer), gcloud and kubectl. Make sure
+    you have all those tools installed.
+
+1.  Make sure that your Composer environment is healthy. Refer to
+    [this documentation](https://cloud.google.com/composer/docs/monitoring-dashboard)
+    for more information specific signals indicating good "Environment health" and
+    "Database health". If your environment is not healthy, fix the environment before
+    running this script.
+
+## Limitations
+
+1.  The script currenlty does not have any error handling mechanism in case of
```

Comment:
```suggestion
1.  The script currently does not have any error handling mechanism in case of
```

Reply:
The script surfaces errors from gcloud when. you do things like pass an incorrect location or environment name, but it also throws a JSON parse error - is this what you mean? (And can it be improved in the future? It's not idea)